docs-kits/kits/Traceability Kit/Software Development View/page_app-provider_software-development-view.mdx

```diff
@@ -46,7 +46,31 @@ The notification states and their cycle are described in the following figure.
 ### Quality Notification API
 A standardized set of APIs and payloads are specified to enable partners to send quality notifications in a standardized way while already knowing which parts (i.e., serialized parts or batches) and which direct customers or suppliers are affected. Notifications are - in contrast to classical data offers in Catena-X which is created for consumption by external stakeholders - a way to push notification related data from a sender to a receiver. For now, this notification API is limited to the sending and receiving of quality notifications as well as the update of the notification status following a state model.
 It is important to mention that the notification API shall be implemented into each participant's traceability solution or solution stack in order to be able to receive information related to quality issues. The implemented endpoints shall be able to process the defined request body and respond with the HTTP status codes and - if required - reply with the defined response body.
-Meanwhile, this notification API shall only be accessible after successful contract negotiation via Connector based on [Dataspace Protocol (DSP)](https://docs.internationaldataspaces.org/dataspace-protocol/), since the API is made available as part of an EDC data asset with usage policy attached. Please refer to the Notification API ([v1.2.1](../openapi/notifications_1-2-1.yaml), [v2.0.0](../openapi/notifications_2-0-0.yaml)) for more details.
+Meanwhile, this notification API shall only be accessible after successful contract negotiation via Connector based on [Dataspace Protocol (DSP)](https://docs.internationaldataspaces.org/dataspace-protocol/), since the API is made available as part of an EDC data asset with usage policy attached. Please refer to the corresponding Notification API specifications for more details:
+- [Notification API (v1.2.1)](../openapi/notifications_1-2-1.yaml)
+- [Notification API (v2.0.0)](../openapi/notifications_2-0-0.yaml)
+
+> :raised_hand: For the current release, **version 1.2.1 is mandatory** and must be supported by every App provider. The newer version 2.0.0 can be supported optionally.
+
+## Block Notifications
+While quality notifications are primarily aimed at (first) contact between business partners (e.g. to initiate a supplier self-assessment), block notifications represent an extended form of data exchange in order to actively initiate an immediate measure to block or sort out the produced parts at the customer's production or logistics. As the quality notifications, block notifications take place between traceability applications or other application stacks, so that this functionality must be implemented by both busniess partners application. However, block notifications include a different status model and, in particular, other information that is critical to the customer's blocking process.
```

Comment:
note: 

- check if the info in brackets is really needed --> (e.g. to initiata asupplier self-assessment)
- typo: ... by both **business** partners application ...
-  "and, in particular, other information that is critical to the customer's blocking process." --> better:
"and, in particular, **additional** information that is critical to the customer's blocking process."

Reply:
@SteDue Done!